Gems/ROS2/Code/Source/Lidar/LidarRaycaster.cpp

```diff
@@ -72,17 +71,11 @@ namespace ROS2
         }
     }
 
-    void LidarRaycaster::ConfigureRayRange(float range)
+    void LidarRaycaster::ConfigureRayRange(RayRange range)
```

Comment:
Any reason to copy the structure? I know it is relatively easy now.

Reply:
I abstained from using references here since the size of the structure is only 8 bytes (since we only support x86_64 linux).